docs/install-and-run.md

```diff
@@ -3,115 +3,126 @@
+And clock on Close.
 
 #### Step 2: Add EarlGrey as a framework dependency
 
-  1. In the test target's section in your `Podfile`, add EarlGrey as a dependency.
-  2. Use the `configure_for_earlgrey` method from the [EarlGrey gem](https://rubygems.org/gems/earlgrey)
-     in your `post_install` actions to add Build Settings and Scheme changes to the **Test Target**
-     for embedding EarlGrey in your app. To use the setup gem, use `gem install earlgrey` before
-     `pod install`.
+1. In the test target's section in your `Podfile`, add EarlGrey as a dependency.
+2. Add EarlGrey as a dependency on your Test Target as in below.
+
+```
+target TEST_TARGET do
+project PROJECT_NAME
+
+use_frameworks! # Required for Swift Test Targets only
+inherit! :search_paths # Required for not double-linking libraries in the app and test targets.
+pod 'EarlGrey'
+end
+```
+
+3. Use the `configure_for_earlgrey` method from the [EarlGrey gem](https://rubygems.org/gems/earlgrey)
+in your Podfile's `post_install` section as in below.
+
+```
+post_install do |installer|
+require 'earlgrey' # Imports the EarlGrey gem for usage in your Podfile
+configure_for_earlgrey(installer, PROJECT_NAME, TEST_TARGET, SCHEME_FILE) # EarlGrey Gem Call
+end
+```
 
-  Use the patten shown in the following Podfile to setup EarlGrey with the latest CocoaPods:
+4. Your complete Podfile when using EarlGrey with the latest CocoaPods should look like:
 
-  ```ruby
-  PROJECT_NAME = 'SampleApp'
-  TEST_TARGET = 'SampleAppTests'
-  SCHEME_FILE = 'SampleAppTests.xcscheme'
+```ruby
+PROJECT_NAME = 'SampleApp'
+TEST_TARGET = 'SampleAppTests'
+SCHEME_FILE = 'SampleAppTests.xcscheme'
 
-  target TEST_TARGET do
-    project PROJECT_NAME
+target TEST_TARGET do
+project PROJECT_NAME
 
-    use_frameworks! # Required for Swift Test Targets only
-    inherit! :search_paths # Required for not double-linking libraries in the app and test targets.
-    pod 'EarlGrey'
-  end
+use_frameworks! # Required for Swift Test Targets only
+inherit! :search_paths # Required for not double-linking libraries in the app and test targets.
+pod 'EarlGrey'
+end
 
-  post_install do |installer|
-    require 'earlgrey' # Imports the EarlGrey gem for usage in your Podfile
-    configure_for_earlgrey(installer, PROJECT_NAME, TEST_TARGET, SCHEME_FILE) # EarlGrey Gem Call
-  end
-  ```
+post_install do |installer|
+require 'earlgrey' # Imports the EarlGrey gem for usage in your Podfile
+configure_for_earlgrey(installer, PROJECT_NAME, TEST_TARGET, SCHEME_FILE) # EarlGrey Gem Call
+end
+```
 
-   * Call the `configure_for_earlgrey` method for each target / Scheme.
-   * For compatibility between different versions, please peruse [this doc](https://github.com/google/EarlGrey/tree/master/docs/versions.md).
-     To download a particular version of the gem, use `gem install earlgrey -v x.y.z`.
+5. Install the EarlGrey gem by doing `gem install earlgrey` and then do the `pod install`.
+
+Notes:
```

Comment:
It doesn't seem like it's possible to add color to plaintext in Github markdown.

http://stackoverflow.com/questions/11509830/how-to-add-color-to-githubs-readme-md-file